src/main/java/com/desiato/puresynth/mappers/DTOMapper.java

```diff
@@ -0,0 +1,20 @@
+package com.desiato.puresynth.mappers;
+
+import com.desiato.puresynth.dtos.UserResponseDTO;
+import com.desiato.puresynth.models.User;
+import org.springframework.stereotype.Component;
+
+
+@Component
+public class DTOMapper {
```

Comment:
Do you want to have a single mapper for all entities? I think it could be easier to maintain to have mapper per controller.